openlibrary/templates/lists/home.html

```diff
@@ -26,7 +26,7 @@ <h2>$_('Create a list of any Subjects, Authors, Works or specific Editions.')</h
     <div class="head">
         <h2>$_('Active Lists')</h2>
     </div>
-    <div id="lists">
+    <div id="user-lists">
```

Comment:
Why is an ID needed for this at all? Since #lists is only used for styling purposes, it would be better if making this change to resort to using a class name.

Reply:
Okay but the ID on the body tag is actually generated and is interconnected in some way. Changing the id breaks the page.

```
class lists_home(delegate.page):
    path = "/lists"

    def GET(self):
        delegate.context.setdefault('bodyid', 'lists')
        return render_template("lists/home")
```
So I don't think we can remove it

Reply:
https://github.com/internetarchive/openlibrary/blob/07e7346293027209f0e75961e7b3d0b0b6e86e1e/openlibrary/templates/site/body.html#L8

The body id is added as an attribute and is no different from a template. We should omit it there. There's no need for a body to have an id - there's only ever gonna be one and it can be accessed via document.body